include/sender_cadmium.hpp

```diff
@@ -56,7 +83,9 @@ class Sender {
             state.model_active     = false;
         }
             
-        // state definition
+        /**
+	* Structure that holds the state variables.
```

Comment:
You may consider providing a brief description to all members (next to it) in the structure.

Reply:
Done